ocaml-variants.opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "ocaml-variants"
+version: "4.08.0+dev"
+synopsis: "OCaml development version"
+depends: [
+  "ocaml" {= "4.08.0" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+]
+conflict-class: "ocaml-core-compiler"
+flags: compiler
+setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make "-j%{jobs}%" "world"]
+  [make "-j%{jobs}%" "world.opt"]
+]
+install: [make "install"]
+maintainer: "caml-list@inria.fr"
+homepage: "https://ocaml.org"
```

Comment:
Given this is the dev version, I think putting the github page here is more appropriate.

Reply:
Fixed this and the other comments above.